packages/debug/src/browser/model/debug-breakpoint.tsx

```diff
@@ -187,8 +187,10 @@ export class DebugBreakpoint extends DebugBreakpointData implements TreeElement
         return <div title={decoration.message.join('\n')} className={classNames.join(' ')}>
             <span className={'theia-debug-breakpoint-icon ' + decoration.className} />
             <input type='checkbox' checked={this.origins[0].enabled} onChange={this.setBreakpointEnabled} />
-            <span className='name'>{this.labelProvider.getName(this.uri)} </span>
-            <span className='path'>{this.labelProvider.getLongName(this.uri.parent)} </span>
+            <span className='line-info' title={this.labelProvider.getName(this.uri) + '    ' + this.labelProvider.getLongName(this.uri.parent)}>
```

Comment:
I'm not sure about the 4 spaces between the `name` and `path`, but perhaps since this is a minor feature the most appropriate implementation would be to do the same as the `explorer` which is to simply provide the `uri` including the `filename`.

```ts
this.labelProvider.getLongName(this.uri)
```

**Example**:
```
java-frontend-module.ts    packages/java/src/browser
```

would become:

```
packages/java/src/browser/java-frontend-module.ts
```

Reply:
Makes sense. Just updated the code